shared/rust/src/api/endpoints/image/recent.rs

```diff
@@ -54,13 +38,15 @@ impl ApiEndpoint for Create {
 /// * ['BadRequest'](http::StatusCode::BAD_REQUEST) if the request is malformed.
 pub struct Update;
 
+// TODO: Move ID into request body
+// TODO: grab req/res from above
 impl ApiEndpoint for Update {
-    type Req = ();
-    type Res = ();
+    type Req = UserRecentImageCreateRequest;
+    type Res = UserRecentImageResponse;
```

Comment:
Rename structs to reflect change in usage here

Reply:
What would be could for names? How about:
Req = UserRecentUpsertRequest
Res = UserRecentImageResponse